### PR TITLE
wormhole: add page

### PR DESCRIPTION
**Pull request by vakili**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by sbrl** [comment]
Good point, @zdroid.

@vakili, do you think we could have a single copy in `common` please? Pages that apply to multiple platforms should go in `common` to avoid duplication. If there are platform-specific differences, we can make an altered copy in that platform's folder, which will override the copy in `common`.

**Comment by zdroid** [comment]
According to https://magic-wormhole.readthedocs.io/en/latest/welcome.html#installation, this works on Windows aswell. Definitely should be in common.

**Comment by vakili** [comment]
Thank you for the feedback. I have moved the page to common.

File: pages/common/wormhole.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# wormhole
+
+> Get things from one computer to another, safely.
+> More information: <https://magic-wormhole.readthedocs.io/en/latest/>.
+
+- Send a file:
+
+`wormhole send {{file}}`
```
Comment (sbrl) [inline pages/common/wormhole.md]: How does it know where to send it? Also, can we tweak this to make it more consistent with other pages please?
```suggestion
`wormhole send {{path/to/file}}`
```

File: pages/common/wormhole.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Send a file:
+
+`wormhole send {{path/to/file}}`
+
+- Receive a file:
+
+`wormhole receive {{wormhole_code}}`
+
+- Send raw text:
```
Comment (agnivade) [inline pages/common/wormhole.md]: Where is the text taken from ?

**Pull request merged**